tests/credential-status/check.no-auth.spec.ts

```diff
@@ -0,0 +1,129 @@
+import {
+    TESTNET_DID,
+    CONTENT_TYPE,
+    PAYLOADS_PATH,
+    NOT_EXISTENT_TESTNET_DID,
+    NOT_EXISTENT_STATUS_LIST_NAME
+} from '../constants';
+import * as fs from 'fs';
+import { test, expect } from '@playwright/test';
+import { StatusCodes } from 'http-status-codes';
+
+test('[Positive] It can check an unencrypted status-list with an existent body and statusPurpose=revocation parameter', async ({ request }) => {
+    const response = await request.post('/credential-status/check?statusPurpose=revocation', {
+        data: {
+            did: TESTNET_DID,
```

Comment:
From my point of view I think it's better to use builders here and in the similar places for other tests.
builder here could be a function, which accepts all the important variables and returns the expected `Request` object.